crates/router/src/configs/defaults.rs

```diff
@@ -4478,10 +4478,217 @@ impl Default for super::settings::RequiredFields {
                                     enums::Connector::Stripe,
                                     RequiredFieldFinal {
                                         mandate: HashMap::new(),
-                                        non_mandate: HashMap::new(),
+                                        non_mandate: HashMap::from([
+                                            (
+                                                "payment_method_data.bank_redirect.eps.billing_details.billing_name".to_string(),
```

Comment:
Mandates have been implemented for this pm type. Could you please confirm?
If implemented, and even mandate req needs this required field, in that case, it should be moved to `common` instead of `non_mandate`

Reply:
[stripe](https://stripe.com/docs/payments/eps),  do not support recurring payments for EPS